spec/components/schemas/Subscription.yaml

```
@@ -137,8 +137,21 @@ properties:
   couponIds:
     type: array
     nullable: true
-    description: A list of coupons to redeem on the customer and restrict to this subscription.
+    description: |
+      A list of coupons to redeem on the customer and restrict to this subscription.
       Read more about [coupons here](https://help.rebilly.com/invoices-and-subscriptions/coupons-discounts).
+
+      This parameter respects the following logic:
+
+      - When not passed then applied coupons will not be changed
```

```suggestion
      - When not passed then applied coupons will not be changed.
```
Let's add punctuation throughout here.

---

Done, thanks